tests/common/regression/test_regression_burn_shares.py

```diff
@@ -0,0 +1,63 @@
+"""
+Tests for lido burnShares method for voting 24/05/2022
+"""
+import eth_abi
+import pytest
+from brownie import reverts, ZERO_ADDRESS, web3, interface, accounts
+from utils.import_current_vote import get_start_and_execute_votes_func
+
+
+start_and_execute_votes = get_start_and_execute_votes_func()
+
+
+@pytest.fixture(scope="module")
+def stranger(accounts):
+    return accounts[0]
+
+
+@pytest.fixture(scope="module")
+def dao_voting_as_eoa(accounts, dao_voting):
+    return accounts.at(dao_voting.address, force=True)
+
+
+@pytest.fixture(scope="module", autouse=(start_and_execute_votes is not None))
```

Comment:
Had a hard times understanding what the actual condition for executing the fixture here. `is not None` check is lacking answers on a Why question.

Reply:
having a variable like `is_there_any_vote_scripts = start_and_execute_votes is not None` will help

Reply:
Or we can return a tuple from the `get_start_and_execute_votes_func()`